src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -695,7 +704,7 @@ public void InvokeCatchingRelevantIOExceptions(Action action, Action<Exception>
             }
         }
 
-        protected virtual ISet<Skimmer<TContext>> CreateSkimmers(TContext context)
+        protected virtual ISet<Skimmer<TContext>> CreateSkimmers(TOptions options, TContext context)
```

Comment:
>TOptions options [](start = 65, length = 16)

do we need this parameter?

Reply:
Good question. We do need this in order to allow for the command-line options to drive skimmer creation. We do this in the SPAM checker. In the future, for example, we might have an argument in our options that points to an assembly that holds skimmers (and the tool will load them dynamically).

---
In reply to: [532125229](https://github.com/microsoft/sarif-sdk/pull/2169#discussion_r532125229) [](ancestors = 532125229)